src/views/VaultDetails.tsx

```diff
@@ -0,0 +1,251 @@
+              shape={ButtonShape.Pill}
+              size={ButtonSize.Small}
+              action={ButtonAction.Navigation}
+              href={vaultsLearnMore}
+              slotRight={<Icon iconName={IconName.LinkOut} />}
+            >
+              {stringGetter({ key: STRING_KEYS.VAULT_FAQS })}
+            </Button>
+          )}
+        </$Buttons>
+      </$Header>
+
+      <$Header>
+        <$DepositWithdrawButtons>
+          <Button
+            type={ButtonType.Button}
+            action={ButtonAction.Primary}
+            shape={ButtonShape.Rectangle}
+            size={ButtonSize.Small}
+            slotLeft={<Icon iconName={IconName.Deposit} />}
+          >
+            {stringGetter({ key: STRING_KEYS.DEPOSIT })}
+          </Button>
+          <Button
+            type={ButtonType.Button}
+            shape={ButtonShape.Rectangle}
+            size={ButtonSize.Small}
+            slotLeft={<Icon iconName={IconName.Withdraw} />}
+          >
+            {stringGetter({ key: STRING_KEYS.WITHDRAW })}
+          </Button>
+        </$DepositWithdrawButtons>
+        <$Details items={items} withSeparators />
+      </$Header>
+    </$VaultDetails>
+  );
+};
+
+const $VaultDetails = styled.div`
+  margin: auto;
+  width: 100%;
+
+  ${layoutMixins.gridConstrainedColumns}
+  --grid-max-columns: 2;
+  --column-gap: 2.25em;
+  --column-min-width: 18rem;
+  --column-max-width: 22rem;
+  --single-column-max-width: 25rem;
+
+  justify-content: center;
+  align-items: center;
+  padding: clamp(0.5rem, 7.5%, 2.5rem);
+  row-gap: 1rem;
+
+  @media ${breakpoints.tablet} {
+    padding: 0 clamp(0.5rem, 7.5%, 2.5rem);
+  }
+`;
+const $Header = styled.header`
+  ${layoutMixins.column}
+  gap: 1.25rem;
+`;
+const $WrapRow = styled.div`
+  ${layoutMixins.row}
+  gap: 0.5rem;
+
+  flex-wrap: wrap;
+`;
+const $MarketTitle = styled.h3`
+  ${layoutMixins.row}
+  font: var(--font-large-medium);
+  gap: 0.5rem;
+
+  img {
+    width: 2.25rem;
+    height: 2.25rem;
+  }
+`;
+const $PositionContainer = styled.div`
+  display: flex;
+  gap: 0.2rem;
+`;
+const $ColoredReturn = styled.div<{ $sign: NumberSign }>`
+  display: flex;
+  gap: 0.25rem;
+  ${({ $sign }) =>
+    $sign &&
+    {
+      [NumberSign.Positive]: css`
+        color: var(--color-positive);
+      `,
+      [NumberSign.Negative]: css`
+        color: var(--color-negative);
+      `,
+      [NumberSign.Neutral]: null,
+    }[$sign]}
+`;
+const $MutedText = styled.span`
+  color: var(--color-text-0);
+`;
+
+const $VaultDescription = styled.div`
+  ${layoutMixins.column}
+  gap: 0.5em;
+
+  p {
+    font: var(--font-small-book);
+
+    &:last-of-type {
+      color: var(--color-text-0);
+    }
+  }
+`;
+const $Buttons = styled.div`
+  ${layoutMixins.row}
+  flex-wrap: wrap;
+  gap: 0.5rem;
+
+  overflow-x: auto;
+`;
+const $DepositWithdrawButtons = styled($Buttons)`
+  justify-content: end;
+`;
+const $Details = styled(Details)`
+  font: var(--font-mini-book);
```

Comment:
<img width="489" alt="Screen Shot 2024-07-01 at 3 01 23 PM" src="https://github.com/dydxprotocol/v4-web/assets/13111994/ef3fbc16-b9b6-4a78-b5f9-35e1fe0dee93">


Comparing to figma, i think we should bump dt from `12px` -> `15px` and dd from `12px` -> `16px`.
Vertical padding is also currently fixed at `8px`, bump to `16px`